frontend/src/pages/modelServing/screens/metrics/MetricsPageTabs.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+import PerformanceTab from './PerformanceTab';
+import BiasTab from './BiasTab';
+import './MetricsPageTabs.scss';
+
+const MetricsPageTabs: React.FC = () => {
+  const DEFAULT_TAB = 'performance';
+
+  const { tab } = useParams();
+  const navigate = useNavigate();
+
+  React.useEffect(() => {
+    if (!tab) {
+      navigate(`./${DEFAULT_TAB}`, { replace: true });
+    }
+  }, [navigate, tab]);
```

Comment:
This didn't work for me, I just end up with a 404 when switching tab, this is why I switched to useEffect. I've just tried this again to verify and, for reasons I don't fully understand - it's still broken this way.

Reply:
Oh, because you're doing relative navigation -- gotcha... yeah, we don't do that anywhere else. What navigation aspects again are you needing with `..` (I don't recall and I'm not checked out on your branch)... I wonder if we could improve those 🤔

Reply:
Ah I see -- okay, let us leave it for now. There is a structured design with our navigation I want to address -- and I think this is one of those moments where we might need to look at how to support holistically throughout our app.

Reply:
Suggested solution for this would be to actually have the wrapper provide you a function that will give you a relative path based on who passed it to you.

I've logged https://github.com/opendatahub-io/odh-dashboard/issues/1169 to handle the infra of making it easier to provide paths.